src/engine.jl

```diff
@@ -113,7 +113,7 @@ function put_variable(session::MSession, name::Symbol, v::MxArray)
     @assert libeng::Ptr{Void} != C_NULL
 
     r = ccall(engfunc(:engPutVariable), Cint,
-        (Ptr{Void}, Ptr{UInt8}, Ptr{Void}), session.ptr, string(name), v.ptr)
+        (Ptr{Void}, Ptr{UInt8}, Ptr{Void}), session.ptr, string(name), v)
```

Comment:
This change is probably not necessary, but probably don't hurt and are a bit safer.